README.md

```diff
@@ -3,11 +3,19 @@ Hood.ie: the Website for the Hoodie Open Source Project
 
 ## Getting started
 
+### Installation
+
 To get started check out the recent version and type `npm install`.
-The default task (just type `grunt`) will fire up a local server at `localhost:1337` with livereload and dev Sass compiling (including sourcemap and nested output).
 
-This doesn't serve the actual jekyll page, though. To do that, run `jekyll serve --watch --drafts` as well, which will run the full page at `localhost:4000`, using the styles from the grunt task.
-You may need to `gem install jekyll-redirect-from jekyll-paginate` beforehand.
+The Hoodie website is built with [Jekyll](https://jekyllrb.com/), Jekyll requires ruby version 2.0.0 or higher.  Open a terminal and type `ruby --version` to check if ruby is installed and up to date.  If not, follow the [official download instructions](https://www.ruby-lang.org/en/downloads/).  Now install [bundler](http://bundler.io/) by running `gem install bundler` on the command line.
```

Comment:
These instructions are paraphrased from the [github pages instructions](https://help.github.com/articles/setting-up-your-pages-site-locally-with-jekyll/#preparing-to-install-the-github-pages-gem).